mmpose/datasets/pipelines/bottom_up_transform.py

```diff
@@ -36,14 +36,16 @@ def _get_multi_scale_size(image,
         - center (np.ndarray)image center
         - scale (np.ndarray): scales wrt width/height
     """
+    assert len(input_size) == 2
```

Comment:
We should support single integer input_size and extend it to [h, w] to avoid bc-breaking. And it's better to mention in the docstring that the order of input_size is [h, w].